packages/brokeneck-react/src/components/Navigation.js

```diff
@@ -6,49 +6,96 @@ import {
   IconButton,
   makeStyles,
   Tab,
-  Tabs,
-  Typography
+  Tabs
 } from '@material-ui/core'
+import { Group as GroupIcon, Person as UserIcon } from '@material-ui/icons'
+
+import MoonIcon from '../icons/moon'
+import SunIcon from '../icons/sun'
 
 import { ThemeSwitcherContext } from './ThemeSwitcherProvider'
 import Provider from './Provider'
 
 const useStyles = makeStyles(theme => ({
   appBar: {
+    alignItems: 'center',
+    backgroundColor: theme.palette.headerBackground.main,
+    color: theme.palette.bodyText.main,
+    flexDirection: 'row',
     marginBottom: theme.spacing(3),
-    flexDirection: 'row'
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(4)
+  },
+  provider: {
+    marginLeft: 'auto'
+  },
+  switcher: {
+    borderLeft: `1px solid ${theme.palette.separator.main}`,
+    borderRight: `1px solid transparent`, // Needed for symetry
+    height: '42px',
+    marginLeft: theme.spacing(2),
+    padding: theme.spacing(2),
+    width: '42px'
   },
-  actions: {
-    marginLeft: 'auto',
-    '& > * + *': {
-      marginLeft: theme.spacing(1)
-    }
+  switcherIcon: {
+    height: 18,
+    width: 18
+  },
+  tabRoot: {
+    minWidth: 120
+  },
+  tabWrapper: {
+    flexDirection: 'row'
```

Comment:
when an element has display: flex (which this one doesn't, btw), direction is row by default. let's avoid setting default values for flexDirection. there are other occurrences of this in this PR, let's remove all of them